dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -62,34 +65,45 @@
   private static void setDetectedOsSystemProperties() {
     // System properties to select Netty dependencies through os-maven-plugin
     // Definition of the properties: https://github.com/trustin/os-maven-plugin
-    String nonAlphaNumericPattern = "[^a-z0-9]+";
+
+    System.setProperty("os.detected.name", osDetectedName());
+
```

Comment:
Do we need this blank line here anymore? Or the next one?

Reply:
Removed them.